client/src/components/app.jsx

```diff
@@ -1,18 +1,48 @@
 import React from 'react';
+import $ from 'jquery';
+import ListingList from './ListingList.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      expanded: false,
+      mockData: {
+        reviews: [],
+      }
     };
+    this.getMockData = this.getMockData.bind(this);
+  }
+
+  componentDidMount() {
+    this.getMockData();
   }
+
+  getMockData() {
+    $.ajax({
+      url: '/reviews/1',
+      type: 'GET',
+      success: (data) => {
+        console.log('data!', data);
+        this.setState({
+          mockData: data,
+        });
+      },
+      error: () => {console.log('error')}
+    });
+  }
+    
+
   render() {
     return (
       <div>
-        WEBPACK SETUP
+        <div>
+          Hello
+        </div>
+        <div>{`${this.state.mockData.reviews.length} Reviews`}</div>
```

Comment:
You're using bat ticks to put js expressions into the html, you could use just curly braces instead. ie
<div>{this.state.mockData.reviews.length} Reviews</div>